rust-toolchain.toml

```diff
@@ -0,0 +1,5 @@
+[toolchain]
+channel = "1.66.1"
+components = [ "rust-src" ]
+targets = [ "wasm32-unknown-unknown", "aarch64-apple-darwin" ]
```

Comment:
Would we not need additional targets for linux?

Reply:
The nice thing about the toolchain file is that is additive, so you'll always get your local system as a target and you'll also get these additional targets